Find a functionally equivalent code:android.widget.TimePicker.setCurrentMinute:COMMENT
Method Modifier: public      
Comment:/**
 * Set the current minute (0-59).
 */

@@ -1,7 +1,3 @@
 {
-    if (currentMinute == getCurrentMinute()) {
-        return;
-    }
-    mMinuteSpinner.setValue(currentMinute);
-    onTimeChanged();
+    mDelegate.setCurrentMinute(currentMinute);
 }

